client/js/lib/d3.js

```diff
@@ -152,9 +153,35 @@ module.exports = function ({
 	}
 
 	function click (d) {
-		const row = document.getElementById(d['hidden-graph-item-id']);
-		if (!row) return;
-		row.classList.toggle('collapsed');
+
+		if(document.querySelector('.filter-table') !== null){
+			const row = document.getElementById(d.name);
```

Comment:
You need: `const row = document.getElementById(d['hidden-graph-item-id']);`

I created this additional hidden property  which contains an amalgamtion of name and source because some rows are from different sources but have the same name.